### ISSUE TITLE
Command timed out = Correct!

### ISSUE THREAD

**Issue by noureddin** [jarv/cmdchallenge#99]
## Summary
`replace_text_in_files` challenge mistakenly accepts a wrong solution.

## Steps to reproduce
In [replace_text_in_files](https://cmdchallenge.com/#/replace_text_in_files), put the following solution (copy and paste).
`a='challenges are difficult';grep -r "$a"|while read b;do sed -i "s/$a//g" "$b";done`

The output should be like this:
````
Command timed out
# 👍 👍 👍  Correct!
# You have a new challenge!
# The file sum-me.txt have a list of numbers,
# one per line. Print the sum of these numbers.
# 
bash(-1)> 
````

This solution is obviously wrong as grep returns filenames and matching lines, which causes problems since sed expects filenames only.

By the way, thanks for the great site!

**Comment by jarv** [comment]
The "command timed out" was due to some server issues last night that should be resolved now, can you try again to see if you are still seeing problems?

**Comment by noureddin** [comment]
Not yet; I have the same output (timed out & Correct) for the same input in this challenge.

By the way, the problem in fact is accepting this incorrect solution, which gives "Command timed out", as correct. I modified the title to clarify this.

As far as I remember, solutions that give "Command timed out" was not considered correct in other challenges.

**Comment by jarv** [comment]
Thanks for the issue, this is fixed in 363d070d666923e9b27558224e5a9238654eb35a
